browser/src/components/NodeSummary.tsx

```diff
@@ -263,16 +254,16 @@ export function NodeSummary({ graphManager, weightService, componentName, nodeNa
         )}
 
         <br />
-        <h6>Dependencies
+       {fullDetails && <h6>Dependencies
```

Comment:
instead adding `fullDetails` check for Dependencies and Callsites separately. Can you just check fullDetails once to determine whether we should show Dependencies and Callsites together, or not